src/tools/illink/test/Mono.Linker.Tests.Shared/AssemblyChecker.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Mono.Cecil;
+
+namespace Mono.Linker.Tests.TestCasesRunner
+{
+	public partial class AssemblyChecker
+	{
+		private readonly AssemblyDefinition originalAssembly;
+
+		static bool IsCompilerGeneratedMemberName (string memberName)
```

Comment:
Maybe we should consider moving these into a separate utils extension method class - they're not really related to the assembly checker as such, they're more general use methods (even if they're not currently used as such).

Reply:
OK - makes sense